libnavigation-router/src/main/java/com/mapbox/navigation/route/onboard/OfflineRoute.kt

```diff
@@ -62,6 +62,7 @@ internal class OfflineRoute private constructor(
         }
     }
 
+    // 'formatWaypointTypes' always returns non-null type
```

Comment:
cc @RingerJK good to make it non-nullable and remove https://github.com/mapbox/mapbox-navigation-android/blob/87ff60774b5e511d6dd54df52ee19ea171b8f7e5/libnavigation-router/src/main/java/com/mapbox/navigation/route/onboard/OfflineRoute.kt#L59-L61

Reply:
LGTM 👍